front-end/src/components/Card/Comment.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { WatchLater } from '@material-ui/icons';
+import { Reply } from './Reply';
+import { GridContainer, GridItem } from '../Grid';
+import { Text } from '../Text';
+import { CreateReplyForm } from '../Forms';
+
+export const Comment = ({ commentId, comments, users }) => {
+  const [showInput, setShowInput] = useState(false);
+
+  return (
+    <GridItem xs={12}>
+      <div
+        style={{
+          paddingTop: '8px',
+          paddingBottom: '8px',
+          paddingRight: '8px',
+          paddingLeft: '30px',
+        }}
+      >
+        <GridContainer>
+          <GridItem xs={12}>
+            <Text variant='body1' style={{ fontWeight: 'bold' }}>
+              {users[comments[commentId].user].name}
+            </Text>
+          </GridItem>
+          <GridItem xs={12}>
+            <Text
+              variant='body2'
+              color='textSecondary'
+              style={{
+                display: 'flex',
+                alignItems: 'center',
+                gap: '5px',
+              }}
+            >
+              <WatchLater style={{ fontSize: 'inherit' }} />
+              {moment(comments[commentId].timestamp).fromNow()}
+            </Text>
+          </GridItem>
+          <GridItem xs={12}>
+            <Text variant='body1' gutterBottom>
+              {comments[commentId].content}
+            </Text>
+          </GridItem>
+          {!showInput && (
+            <GridItem xs={12}>
+              <Text
+                variant='body2'
+                color='textSecondary'
```

Comment:
define all colors in the theme object and pass color using that.

Reply:
The color is actually a prop which is already defined in the default theme provided by Material UI. Material UI has been used as the component UI library.